src/extra.i

```diff
@@ -3536,6 +3538,14 @@ void JM_make_image_block(fz_stext_block *block, PyObject *block_dict)
             buf = buffer->buffer;
         } 
         bytes = JM_BinFromBuffer(buf);
+        if (image->mask) {
+            fz_buffer *mask = fz_new_buffer_from_image_as_png(ctx, image->mask, fz_default_color_params);
+            mask_bytes = JM_BinFromBuffer(mask);
+            fz_drop_buffer(ctx, mask);
```

Comment:
We should use `mupdf::fz_new_buffer_from_image_as_png()` to get a FzBuffer instead of a fz_buffer*, so avoiding the explicit call to `fz_drop_buffer`, and avoiding a leak if JM_BinFromBuffer() raises an exception. [JM_BinFromBuffer() is overloaded to take a fz_buffer* or a FzBuffer.]